makelib/golang.mk

```diff
@@ -226,12 +235,12 @@ go.generate:
 # Common Targets
 
 build.init: go.init
-build.check: go.fmt
-build.check.platform: go.vet
+build.check: go.lint
```

Comment:
I'm a bit confused why you dropped go.fmt and go.vet in this change.

Reply:
@bassam Please take a look at the PR description - I've covered that there. In short, they're redundant.

Reply:
doh. ok, awesome.